pages/events/fall-fest.vue

```diff
@@ -110,7 +112,7 @@ const scheduleToTableData = (slot: eventDetails) => [
 ];
 
 const agenda = {
-  title: "Qiskit Fall Fest Extension Events",
+  title: "Qiskit Fall Fest: Extension Events",
```

Comment:
Here is where you would define the `description`

```suggestion
  title: "Qiskit Fall Fest: Extension Events",
  description: "Extension Events are intimate gatherings at various campuses, tailored for a select few attendees. Due to their close-knit nature, we do not provide links as these events primarily take place within the institution's community.",
```

Reply:
resolved in https://github.com/Qiskit/qiskit.org/pull/3550/commits/9e7d698cbf2a0f898bd132f2575590cc22107a32